degov_node_normal_page/templates/node--normal_page--slideshow.html.twig

```diff
@@ -90,7 +90,7 @@
 
   {% if content.field_teaser_image %}
     <a href="{{ url }}" rel="bookmark">
-      <div class="normal-page__slideshow-image slideshow-image {{ class_sidebar }} arrow">
+      <div class="normal-page__slideshow-image slideshow-image slide__media {{ class_sidebar }} arrow">
```

Comment:
This is needed to have unified method to get the previous and next image for the slideshow type 2 thumbnails